eop.c

```diff
@@ -20,8 +20,9 @@ static void signalHandler(int signalNumber)
 void usage(const char *program)
 {
 	fprintf(stderr, "Usage: %s ", program);
-	fprintf(stderr, "<file> [x y w h]\n");
+	fprintf(stderr, "<file> l [x y w h]\n");
```

Comment:
The `[]` here denotes optional param so I think the layer should also be denoted as optional `> [l] [`